rector-ci.php

```diff
@@ -61,4 +61,5 @@
 
     # so Rector code is still PHP 7.2 compatible
     $parameters->set(Option::PHP_VERSION_FEATURES, '7.2');
+    $parameters->set(Option::ENABLE_CACHE, true);
```

Comment:
@TomasVotruba Not sure here, we can turn that on, but dead code is being skipped.

Reply:
Sure we can :) 
I'd make it `false` by default before merging.

How about warning people in command line about the `dead code` being skipped on cache?

Reply:
There already is warning, but it just says number of rectors skipped, do you want to warn with list of rectors being skipped?

Reply:
It is false by default even if I just remove it, should I rather keep it with false or just remove it?

Reply:
Ah, great :)  I confused this with internal config. All good then :+1: